app/containers/ActionEdit/index.js

```diff
@@ -73,23 +68,35 @@ export class ActionEdit extends React.PureComponent { // eslint-disable-line rea
           </div>
         }
         {action &&
-          <Form
+          <EntityForm
             model="actionEdit.form.action"
-            onSubmit={this.props.handleSubmit}
-          >
-            <label htmlFor="title">Title:</label>
-            <Control.text id="title" model=".title" />
-            <label htmlFor="description">Description:</label>
-            <Control.textarea id="description" model=".description" />
-            <label htmlFor="status">Status:</label>
-            <Control.select id="status" model=".draft" value={action && action.draft}>
-              {PUBLISH_STATUSES.map((status) =>
-                <option key={status.value} value={status.value}>{status.label}</option>
-              )}
-            </Control.select>
-            <button type="submit">Save</button>
-            </Form>
-          }
+            handleSubmit={this.props.handleSubmit}
+            fields={
+              [
+                {
+                  id: 'title',
+                  label: 'Title',
+                  type: 'text',
+                  model: '.title',
+                },
```

Comment:
We could have sensible defaults for many of these I think, i.e. default to Input type, default label to be Caps case of id, default model to be `.${name}`

In which case you could just pass `{ id: 'title' }` here

It might be better to keep it more verbose like this, so that its clear tho.